package.json

```diff
@@ -43,6 +43,7 @@
     "react-router": "^4.3.1",
     "react-router-dom": "^5.1.2",
     "react-router-redux": "^5.0.0-alpha.9",
+    "react-transition-group": "^4.4.1",
```

Comment:
are you using this library in the code changes? I did not see it or know where to look sorry

Reply:
I'm not, but I couldn't compile the code without it, so it seemed like a requirement. I can move this commit to a separate PR, if that seems like the thing to do.

Reply:
oh wow let me check this one, that is odd

Reply:
interesting, so I locally uninstalled the library npm uninstall react-transition-group then did a npm ci to ensure deps are all good in lock file, then compiled, and it starts for me, on your branch ...

maybe try deleting your node_modules, then doing above?

if works we can try removing in the next commit and see if it passes the build

Reply:
Removed!